conf/log4j.properties

```diff
@@ -26,6 +35,15 @@ log4j.appender.WORKER_LOGGER.layout=org.apache.log4j.PatternLayout
 log4j.appender.WORKER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{2} (%F:%M) - %m%n
 #log4j.appender.WORKER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{2} (%F:%M(%L)) - %m%n
 
+# Appender for Worker Access Log
```

Comment:
I'm curious. Do you foresee the access logs for the worker and master to be different formatting?